src/java.base/share/classes/java/io/Reader.java

```diff
@@ -140,6 +140,208 @@ public void close() {
         };
     }
 
+    /**
+     * Returns a new {@code Reader} whose source is a {@link CharSequence}.
+     *
+     * <p> The returned reader's class provides an API compatible with
+     * {@link StringReader}, but with no guarantee of synchronization, and
+     * without limitation to {@code String} sources. This class is designed for
+     * use as a drop-in replacement for {@code StringReader} in places where the
+     * reader was being used by a single thread (as is generally the case).
+     * Where possible, it is recommended that this class be used in preference
+     * to {@code StringReader} as it will be faster under most implementations.
+     *
+     * <p> The returned stream is initially open. The stream is closed by
+     * calling the {@code close()} method. Subsequent calls to {@code close()}
+     * have no effect.
+     *
+     * <p> After the stream has been closed, the {@code read()},
+     * {@code read(char[])}, {@code read(char[], int, int)},
+     * {@code read(CharBuffer)}, {@code ready()}, {@code skip(long)}, and
+     * {@code transferTo()} methods all throw {@code IOException}.
+     *
+     * <p> The {@code markSupported()} method returns {@code true}.
```

Comment:
```suggestion
     * <p>The returned reader supports the {@link #mark mark()} operation.
```

Reply:
This is not wrong. This style is used all over the place already within `Reader.java`, so it would be strange to change it *just here*.

Reply:
You can “elaborately” the close and mark protocol in the Reader interface and then only mention itnadgeres to it here.